reactor-core/src/test/java/reactor/core/scheduler/ParallelSchedulerTest.java

```diff
@@ -40,16 +41,37 @@ protected Scheduler scheduler() {
 		return Schedulers.newParallel("ParallelSchedulerTest");
 	}
 
-	@Test(expected = IllegalArgumentException.class)
-	public void negativeParallelism() throws Exception {
-		Schedulers.newParallel("test", -1);
-	}
-
 	@Override
 	protected boolean shouldCheckInterrupted() {
 		return true;
 	}
 
+	@Test
+	public void startAndDecorationImplicit() {
```

Comment:
this test seems to be duplicated multiple times. Can we reuse it?

Reply:
these tests are not 100% the same, as parallel and elastic have different implicit start behaviors: one starts a single executor service on demand (as a worker gets picked), while the other lazily starts its executor services (ALL of them) on first task submission